php-practice/section13.php

```diff
@@ -67,6 +73,11 @@ class TwelveSidedDie extends Dice {
  * Echo out the PREFIX from your ErrorOutputter class
  */
 
+$errorOutputter = new ErrorOutputter();
+
+$errorOutputter->outputError('Syntax Error');
+
+echo SEPARATOR;
```

Comment:
Actually the intention for this problem was to have you echo out the `PREFIX` constant from your `ErrorOutputter` class.

e.g.

```php
echo TwelveSidedDie::SIDES; // Echos out 12 to the screen
```